corehq/apps/repeaters/migrations/f_0001_adjust_auth_field_format.py

```diff
@@ -0,0 +1,37 @@
+from django.db import migrations
+
+from corehq.apps.repeaters.models import Repeater
+from corehq.sql_db.operations import HqRunPython
+from corehq.util.couch import iter_update, DocUpdate
+
+
+def migrate_auth_field(apps, schema_editor):
+    repeater_ids = [row['id'] for row in Repeater.view(
+        'receiverwrapper/repeaters',
+        include_docs=False,
+        reduce=False,
+        wrap_doc=False
+    )]
```

Comment:
do you know how many repeaters this will need to update? i'm not sure if we'd want to block deploy on it if it will take a long time